src_Core/Core/CoreW.bsv

```diff
@@ -423,78 +484,118 @@ module mkCoreW #(Reset dm_power_on_reset)
    endrule
 
    // ================================================================
-   // INTERFACE
-
-   // ----------------------------------------------------------------
-   // Debugging: set core's verbosity, htif addrs
+   // Connect external debug module interface
 
-   method Action  set_verbosity (Bit #(4)  verbosity, Bit #(64)  logdelay);
-      // Warning: ignoring logdelay
-      proc.set_verbosity (verbosity);
-   endmethod
+   let dbgShim <- mkAXI4LiteShim (reset_by porReset);
 
-   // ----------------------------------------------------------------
-   // Start
-
-   method Action start (Bool is_running, Bit #(64) tohost_addr, Bit #(64) fromhost_addr);
-      plic.set_addr_map (zeroExtend (soc_map.m_plic_addr_range.base),
-                         zeroExtend (rangeTop(soc_map.m_plic_addr_range)));
-
-      let pc = soc_map_struct.pc_reset_value;
-      proc.start (is_running, pc, tohost_addr, fromhost_addr);
-
-`ifdef INCLUDE_GDB_CONTROL
-      // Save for potential future use by rl_dm_harts_reset
```

Comment:
Did we lose this functionality? If so, is there no need for this anymore?

Reply:
There is just a reset now, not a start method. and reseting, well.. restarts... (at least it should if there are no bugs :p)